tests/unit/util/test_version_util.py

```diff
@@ -112,31 +112,20 @@ def test_trailing_non_numeric(self):
     def test_ancient_tags(self):
         # Teradata and non-Teradata versions
         self.assertEqual(
-            (0, '97t'), strip_tag(('0', '97t-SNAPSHOT')))
+            (0, '97t'), strip_tag(('0', '97t', 'SNAPSHOT')))
         self.assertEqual(
-            (0, 99), strip_tag(('0', '99-SNAPSHOT')))
+            (0, 99), strip_tag(('0', '99', 'SNAPSHOT')))
 
     def test_teradata_version(self):
         self.assertEqual(
             (0, '115t'), strip_tag(('0', '115t')))
         self.assertEqual(
             (0, '123t'), strip_tag(('0', '123t', 'SNAPSHOT')))
 
-    def test_non_trailing_non_numeric(self):
-        self.assertRaises(
-            ValueError, strip_tag, ('1', 'TWO', '3'))
```

Comment:
Test for new behavior specified by comment here: https://github.com/prestodb/presto-admin/pull/305/files#diff-c6d431ed89d19e8713c881bbf37a396aR58